scripts/clean_build.sh

```diff
@@ -11,9 +11,7 @@ rm dx*.jar
 # Clean artifacts from prev. builds, platform
 username=`dx whoami --id`
 dx rm -r dxCompiler_playground:/builds/$username
-dx rm -r dxCompiler_playground:/builds/user-$username
```

Comment:
`dx whoami` with `--id` already returns the user-xxxx form of the username.

Looking at the implementation of dxApi.whoami(), it looks consistent with the dxpy one (both use the same system API call), so the unit_tests folder should also get the same name.